### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by sjd78**
Update the documentation to make it consistent with the current state of the project.

### CODE REVIEW

**Comment by ovirt-infra** [comment]
All tests passed

File: doc/releaseProcess.md [hunk 1]
```diff
@@ -1,37 +1,40 @@
...
-- at https://github.com/oVirt/ovirt-web-ui/releases create new release:
+- At https://github.com/oVirt/ovirt-web-ui/releases create new release:
   - release name (tag): 1.5.2
   - title: ovirt-web-ui-1.5.2 (27-Mar-2019)
   - description: copy & paste from spec file
 
-- create releng-tools patch for ovirt compose. Example: https://gerrit.ovirt.org/#/c/90640/
+- Update oVirt compose with the new release
 
 - Docker builds
```
Comment (rszwajko) [inline doc/releaseProcess.md:39]: docker builds are still supported?

File: doc/releaseProcess.md [hunk 2]
```diff
@@ -1,37 +1,40 @@
...
 
-  - git add -u
-  - git commit
-    - "ovirt-web-ui 1.5.2"
-  - git push --set-upstream origin release-1.5.2
-- open pull request, wait for CI, merge it
+- Open pull request from your new branch, wait for CI, merge it
 
-- at https://github.com/oVirt/ovirt-web-ui/releases create new release:
+- At https://github.com/oVirt/ovirt-web-ui/releases create new release:
```
Comment (rszwajko) [inline doc/releaseProcess.md:32]: The link is not clickable

File: doc/releaseProcess.md (hunk 1 above)
Comment (sjd78) [reply doc/releaseProcess.md:39]: Well, the Dockerfile looks mostly ok (node version needs to be updated) so it is possible.  I'd rather keep it at this point.

File: DEVELOPERS.md [hunk 3]
```diff
@@ -15,38 +15,43 @@ Please report bugs and feature requests to the [GitHub issue tracker](https://gi
...
 #### ovirt-engine packages
-Install `ovirt-engine-nodejs-modules` from the `ovirt/tested` yum repo for your platform
-to use the same packages that will be used by CI to build the app.
+Install `ovirt-engine-nodejs-modules` from the [ovirt/ovirt-master-snapshot copr repo](https://copr.fedorainfracloud.org/coprs/ovirt/ovirt-master-snapshot/)
+for your platform to use the same packages that will be used by CI to build the app.
 
-    REPO=fc30 # or the appropriate release and version for you
-    dnf config-manager --add-repo http://resources.ovirt.org/repos/ovirt/tested/master/rpm/$REPO
-    dnf install ovirt-engine-nodejs-modules
+For CentOS stream 9:
```
[the remark this answers is not in the document]
Comment (sjd78) [reply DEVELOPERS.md:30]: Humm.  I'll leave it in but add a few comments to help.

**Comment by ovirt-infra** [comment]
All tests passed

**Review by rszwajko — APPROVED** [review]

**Pull request merged**